test/unit-test/dawn/Optimizer/Passes/samples/test_compute_maximum_extent_02.cpp

```diff
@@ -0,0 +1,35 @@
+//===--------------------------------------------------------------------------------*- C++ -*-===//
```

Comment:
should this forlder contain a qt mock to include?

Reply:
what's that?

Reply:
we have these in the GTClang projects:
```
# Add a dummy target with all the source files so that they are included in a qt creator project
mchbuild_add_custom_dummy_target(NAME GTClangQtCreatorParseProjectCodeGenTest
                         DIRECTORIES ${cwd}/CodeGen)
```